extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/RasterDemSource.kt

```diff
@@ -242,6 +242,52 @@ class RasterDemSource(builder: Builder) : Source(builder.sourceId) {
      */
     get() = getPropertyValue("max-overscale-factor-for-parent-tiles")
 
+  /**
+   * For the tiled sources, this property sets the tile requests delay. The given delay comes in
+   * action only during an ongoing animation or gestures. It helps to avoid loading, parsing and rendering
+   * of the transient tiles and thus to improve the rendering performance, especially on low-end devices.
+   */
+  fun tileRequestDelay(value: Long = 1L) = apply {
```

Comment:
we should generate `Double` type(gl-native type is float)

Reply:
As discussed offline, will update this to `Double` type. 👍